pandas/core/groupby/generic.py

```diff
@@ -262,10 +262,9 @@ def aggregate(self, func=None, *args, **kwargs):
 
             try:
                 return self._python_agg_general(func, *args, **kwargs)
-            except (AssertionError, TypeError):
+            except (AssertionError, TypeError, IndexError):
```

Comment:
is there a test change?

Reply:
No, but we currently have tests that raise IndexError that no longer will